spec/support/repo/Gemfile

```diff
@@ -19,3 +19,5 @@ gem("smart_properties")
 gem("activesupport")
 # Needed to test Git gems
 gem("ast", git: "https://github.com/whitequark/ast", ref: "e07a4f66e05ac7972643a8841e336d327ea78ae1")
+# Needed to test missing gems
+gem("nokogiri", ">= 1.11.1")
```

Comment:
Not sure if the 2.4 failures are related or not: https://github.com/Shopify/tapioca/pull/213/checks?check_run_id=1765659365 maybe a conflict with the `ast` version required the line before?

Maybe the command logged the failures in a file somewhere and we can check 🤯

Reply:
Yeap, indeed that was the problem:
```
$ bundle
Fetching https://github.com/whitequark/ast
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Bundler could not find compatible versions for gem "ruby":
  In Gemfile:
    ruby

    nokogiri (>= 1.11.1) was resolved to 1.11.1, which depends on
      ruby (>= 2.5.0)

    tapioca was resolved to 0.4.13, which depends on
      ruby (>= 2.4)
```
I will need to skip this dependency on Ruby 2.4. At least the test won't break since it tests the absence of the RBI generation 😅